doc/source/getting_started/installation.rst

```diff
@@ -16,7 +16,7 @@ Xorbits can be installed via pip from `PyPI <https://pypi.org/project/xorbits>`_
 Python version support
 ----------------------
 
-Officially Python 3,7, 3.8, 3.9 and 3.10.
+Officially Python 3.8, 3.9 and 3.10.
```

Comment:
I forgot to mention that Xorbits supports python 3.11.

https://github.com/xprobe-inc/xorbits/blob/da289d997ff966773027143bcfe0257d9ab73dd1/.github/workflows/python.yaml#L75